### ISSUE TITLE
Identyfikator płatności

### ISSUE THREAD

**Issue by Bober95** [allegro/allegro-api#7315]
Witam, czy poprawnym zachowaniem jest to iż płatność do zamówienia wpada w pierwszej kolejności jako P24 którego klient nie opłaca, następnie po chwili opłaca to zamówienie ale już przez PayU, ale nie następuje zmiana identyfikatora płatności.

**Comment by PrzemyslawLukanowski** [comment]
Tak, to poprawne zachowanie i nie należy tego traktować jako błąd. Istotne są dane, które otrzymasz dla statusu READY_FOR_PROCESSING, ponieważ wtedy wiesz, że zamówienie jest opłacone i możesz przejść do jego realizacji.

**Comment by Bober95** [comment]
No właśnie dla nas później ten typ płatności jest istotny dla rozliczeń. Ale ok, poprawię we własnym zakresie aby po za identyfikatorem sprawdzało również usługodawcę tej płatności.
